src/coreclr/jit/valuenum.cpp

```diff
@@ -8266,15 +8266,15 @@ ValueNum ValueNumStore::EvalMathFuncUnary(var_types typ, NamedIntrinsic gtMathFN
                     break;
 
                 case NI_System_Math_Acos:
-                    res = acos(arg0Val);
+                    res = netintrinsics_acos(arg0Val);
                     break;
 
                 case NI_System_Math_Acosh:
                     res = acosh(arg0Val);
```

Comment:
Was acosh missed? Should we be making them all consistent?

Reply:
I only moved over the APIs that currently existed in the PAL today. All others didn't have any special shimming over the CRT on any platform.